level.go

```diff
@@ -1,18 +1,44 @@
 package engo
 
+import (
+	"engo.io/gl"
+)
+
 type Level struct {
-	Width      int
-	Height     int
+	width      int
```

Comment:
Yeah I am going to redo a bunch of the level stuff, the code changes to level and tile right now are just so that it's compliant with the current ECS implementation